src/components/TasksList/TabPanel.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: any;
+  value: any;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <Box hidden={value !== index} {...other} p={3}>
+      {children}
+    </Box>
+  );
+}
+
+export default TabPanel;
```

Comment:
Are you sure that this component shouldn't be located inside `/src/TabPanel/` instead `/src/TaskList` folder ?  (I know it's a part of TaskList, but we have other components that are parts of other views and still those components are located in the `/src/**`) 🤔

Reply:
Not really since this component doesn't live alone. it needs TaskList

Reply:
TabPanel component added to component folder.